src/main/java/com/petitapetit/miml/domain/member/model/Member.java

```diff
@@ -1,27 +1,41 @@
-package com.petitapetit.miml;
+package com.petitapetit.miml.domain.member.model;
 
 import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.Table;
 
+import com.petitapetit.miml.domain.auth.oauth.provider.OAuth2Provider;
+
 import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Entity
 @Getter
 @AllArgsConstructor
 @NoArgsConstructor
+@Builder
 @Table(name = "member_tb")
 public class Member {
 
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Long id;
-	private String spotifyId;
+	// private String spotifyId;
 	private String name;
 	private String email;
-	private String profileImage;
+	private String image;
+
+	@Enumerated(EnumType.STRING)
+	private RoleType role;
+
+	@Enumerated(EnumType.STRING)
+	private OAuth2Provider provider;
+
+	private String providerId;
```

Comment:
`NIT`: 해당 정보들은 따로 묶는것도 좋을 것 같습니다!
```suggestion
	@Enumerated(EnumType.STRING)
	private OAuthInfo oAuthInfo;
	
public class OAuthInfo {
    private OAuth2Provider provider;
    private String providerId;
}
```

Reply:
좋습니다!! 해당 코멘트에서 인사이트를 얻어서 복합키를 적용해 보았는데요, `provider`와 `providerId`를 묶어 사용자를 식별하는데 사용합니다. 좋은 리뷰 감사합니다! 👍